### ISSUE TITLE
get rid of pygit2

### ISSUE THREAD

**Issue by paultag** [paultag/revu#16]
once dulwich works on python 3

**Comment by NathanW2** [comment]
Maybe using `sh` would be easy if it's going to be wrapped in a class anyway.  Installing pygit seems to be a bit of a bugger.

**Comment by paultag** [comment]
@NathanW2 Yeah, the only hangup is it's really slow; like, really really slow.

I super want a nativeish binding for handling stuff like per-file log things, and getting bits of information on the tree

**Comment by NathanW2** [comment]
Bugger. Didn't know it was slow.
